### PR TITLE
Fixes gh-1754 Change a warning re calling certain Lua functions from SQL

### PR DESCRIPTION
**Pull request by pgulutzan**

### CODE REVIEW

File: doc/reference/reference_sql/sql_plus_lua.rst [hunk 1]
```diff
@@ -404,11 +404,12 @@ For example this will show a database configuration member: |br|
 For example this will return FALSE because Lua nil and box.NULL are the same as SQL NULL: |br|
 ``box.execute([[SELECT lua('return box.NULL') IS NOT NULL;]])``
 
-Warning: the SQL statement must not invoke a Lua function, or execute a Lua chunk,
+Warning: the SQL statement should not invoke a Lua function, or execute a Lua chunk,
 that accesses a space that underlies any SQL table that the SQL statement accesses.
 For example, if function ``f()`` contains a request ``"box.space.TEST:insert{0}"``,
 then the SQL statement ``"SELECT f() FROM test;"`` will try to access the same space in two ways.
-The results of such conflict may include a hang or an infinite loop.
+For Tarantool version 2.6 or earlier the results of such conflict may include a hang or an infinite loop.
```
Comment (NickVolynkin) [inline doc/reference/reference_sql/sql_plus_lua.rst:411]: Is it 2.6.1 or 2.6.2 we're speaking about?

File: doc/reference/reference_sql/sql_plus_lua.rst [hunk 2]
```diff
@@ -404,11 +404,12 @@ For example this will show a database configuration member: |br|
...
+For Tarantool verrsion 2.7 or later the results may be correct but are not guaranteed.
```
Comment (NickVolynkin) [inline doc/reference/reference_sql/sql_plus_lua.rst:412]:
```suggestion
For Tarantool version :doc:`2.7.1 </release/2.7.1>` or later the results may be correct but are not guaranteed.
```

**Comment by NickVolynkin** [comment]
As https://github.com/tarantool/doc/issues/1754 is closed, should we close this PR as well?

**Comment by pgulutzan** [comment]
Yes, I should have thought of closing the pull request at the same time as I closed the issue.

**Pull request closed without merging**